autoendpoint/Cargo.toml

```diff
@@ -45,7 +45,7 @@ tokio.workspace = true
 url.workspace = true
 uuid.workspace = true
 
-a2 = {version = "0.8", git = "https://github.com/mozilla-services/a2"}
+a2 = {version = "0.8", git = "https://github.com/mozilla-services/a2", branch="master"}
```

Comment:
I was thinking more of a specific sha (thinking of reproducable builds) but then again we weren't doing this before by pointing to the autoendpoint branch, so never mind I guess.